### ISSUE TITLE
When adding a new task, can a due date be set?

### ISSUE THREAD

**Issue by buckmaxwell** [sachaos/todoist#71]
I'm sorry, perhaps this is obvious. Is there a way to set a due date on tasks added through the cli?

**Comment by hrmJ** [comment]
This is unclear for me, too. If I just write "next week", it doesn't seem to get parsed to a date

**Comment by aleksandar-todorovic** [comment]
Yes, use `todoist <command> help` to see the flags (in this case, `todoist add help`).

In short: `todoist add "This is a test" -N "Personal" -d "next week" -L "test"` will add a task in the project named "Personal" that's due next Monday and has a label "test".

So, for your specific question, instead of doing `todoist add "This is a test tomorrow"`, do `todoist add "This is a test" -d "tomorrow"`.

**Comment by buckmaxwell** [comment]
thanks!
